### PR TITLE
Added validation results and new material parameter set, by BUW-FZJ.

### PR DESCRIPTION
**Pull request by TristanHehnen**
Hi,

as the title says. 
A new PMMA material parameter set is provided.
For validation, simulations of the NIST Gasification Apparatus are conducted and their results provided.

Simulation in and output is available from a [Zenodo](https://zenodo.org/record/8005836) repository.

Best,
Tristan

### CODE REVIEW

**Comment by leventon** [comment]
In general, this looks good/results are well formatted. I am a little confused by the reported 14 to 15 digit precision in sample mass / temperature. This suggests an accuracy in model results that I'm not sure is intended.

For your review, a comment on boundary conditions is is included directly in the README file. 

Also can you confirm your property set naming convention? 
2021, MaCFP-2: BUW-FZJ-A and BUW-FZJ-B
2023, MaCFP-3: BUW-FZJ-PMMA01 BUT this is *not* the same PMMA01 property set as used in reference [3]. And, perhaps, PMMA01 is similar to Approach A (based on the link provided in the README)?

**Comment by TristanHehnen** [comment]
Sure, I can cut it down to four decimal places.

I don't quite understand what you mean with "... a comment on boundary conditions is is included directly in the README file.". Could you please elaborate on what you do want me to add/change?

Yes, PMMA01 is different than the other two (approach A/B).
Yes, it is NOT from reference [3], it is based on lessons learned from it and built on a point made in the discussion (species being released that are inert or combustible thus mixing on-the-fly to achieve the observed effective heat of combustion from the experiment). I will just remove this reference. 
I'm not sure how to judge similarity here. All three parameter sets are reproducing the cast black PMMA from MaCFP, so they are similar. All use different strategies to determine the material and decomposition model parameters, with PMMA01 being fundamentally different, so they are not similar. What specifically do you want me to add/change here?

File: PMMA/Validation_Results/BUW-FZJ/README.md
Comment (TristanHehnen) [inline PMMA/Validation_Results/BUW-FZJ/README.md:1]: Somehow I do not see a comment here. Did I do something wrong?

**Comment by leventon** [comment]
regarding the naming convention, could we consider naming your third model as 'C', such that we continue on in series from BUW-FZJ-A and BUW-FZJ-B to BUW-FZJ-C?

Sorry for the confusion, I maybe misread the README. it sounded like PMMA01 was also the name of the model you developed in ref [3] but that your third pyrolysis model submitted to MaCFP-3 (which is not the same property set) was now also called PMMA01. For continuity, It'd be clearest to me if you had three models a,b,c (like DBI has 1, 2, 3, 4; rather than 1 2 3 X).

Though I think it is easier to understand (changing the name of your third model BUW-FZJ-PMMA01 to BUW-FZJ-C) I will not _force_ this change; the immediate issue/confusion to me was having two different models both ID'd in the README with the same link to the same .json file and not being sure if PMMA01 here was the same name used for PMMA01 in [3]. With an update to the README, you'll have addressed these concerns.

**Comment by TristanHehnen** [comment]
I changed the naming and adjusted the link.

**Comment by leventon** [comment]
Thank you, Tristan.

**Comment by TristanHehnen** [comment]
Thanks!

**Pull request merged**